api/snapshot.go

```diff
@@ -186,6 +186,14 @@ func (s *Server) SnapshotBackup(w http.ResponseWriter, req *http.Request) (err e
 		return err
 	}
 
+	backupMode := input.BackupMode
+	if backupMode == "" {
+		backupMode = string(longhorn.BackupModeIncremental)
+	}
+	if err := util.ValidateBackupMode(backupMode); err != nil {
+		return err
+	}
+
```

Comment:
Can we remove the validation and move it to the validating webhook?

Reply:
We do 
- add `backupMode` as `incremental` when it is empty
- validate if `backupMode` is empty
in our webhook, we can remove this code snippet here